### PR TITLE
Accounts-google reorganize

### PR DESCRIPTION
**Pull request by laosb**
#7715
Related (the Facebook one): #7728 

Hi there, long time no see!

Having my winter vacation. Thinking accounts-google is popular, I reorganized accounts-google this time. I haven't got some time to test it, so getting someone to test it would be helpful.

I can't guarantee when I can do this on accounts-twitter, accounts-github and others. Pull requests for those are always encouraged.

### CODE REVIEW

**Comment by laosb** [comment]
Travis CI said:

```
An error occurred while generating the build script.
```

Don't know what's happening.

File: packages/google/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# google
+# google-oauth
 [Source code of released version](https://github.com/meteor/meteor/tree/master/packages/google) | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/google)
 ***
 
-An implementation of the Google OAuth flow. See the [project page](https://www.meteor.com/accounts) on Meteor Accounts for more details. XXX link
\ No newline at end of file
+** Deprecated, use facebook-oauth instead**
```
Comment (abernix) [inline packages/google/README.md]: This should be `google-oauth`.

File: packages/google-oauth/package.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  summary: "Google OAuth flow",
+  version: "1.2.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use(['underscore', 'service-configuration'], ['client', 'server']);
+  api.use(['random', 'templating@1.2.13'], 'client');
```
Comment (abernix) [inline packages/google-oauth/package.js]: Is there a particular reason we need to depend on not only `templating` but specifically `templating@1.2.13`?  The `facebook-oauth` package doesn't use `templating` at all.

File: packages/google-config-ui/package.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: "Blaze configuration templates for Google OAuth.",
+  version: "1.0.0-beta.1"
+});
+
+Package.onUse(function(api) {
+  api.use('templating@1.2.13', 'client');
```
Comment (abernix) [inline packages/google-config-ui/package.js]: Was there a reason for depending specifically on `1.2.13`?  We do this in `facebook-config-ui` but I don't immediately remember why.

File: packages/google-config-ui/package.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: "Blaze configuration templates for Google OAuth.",
+  version: "1.0.0-beta.1"
```
Comment (abernix) [inline packages/google-config-ui/package.js]: Was there a reason to `-beta.1` this?

File: packages/google-oauth/package.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+  api.export('Google');
```
Comment (abernix) [inline packages/google-oauth/package.js]: Nit pick, but this should be after `api.addFiles` just to match the [Facebook package](https://github.com/meteor/meteor/blob/a4dad54a097f3630a1e583865044be5823e13621/packages/facebook-oauth/package.js)

File: packages/google/README.md (hunk 1 above)
Comment (laosb) [inline packages/google/README.md]: Right, my copying issue.

File: packages/google-config-ui/package.js (hunk 4 above)
Comment (laosb) [inline packages/google-config-ui/package.js]: That's copied from `facebook-config-ui`.

File: packages/google-config-ui/package.js (hunk 3 above)
Comment (laosb) [inline packages/google-config-ui/package.js]: I guess this was added for the release version pinning?

**Pull request merged**